common-v2/jquery-2.2.4.js

```diff
@@ -319,32 +319,6 @@ jQuery.extend( {
 			typeof obj;
 	},
 
-	// Evaluates a script in a global context
```

Comment:
The common-v2/jquery-2.2.4.js appears to be original jQuery 2.2.4, why do you need to modify it? Doing so would change the file fingerprint and could cause warnings if package gets scanned

Reply:
For that change, my intention is to make BE get rid of the warning message of using 'eval' on Firefox. Never mind, after discussing with Mr. @dluong-pi , we have concurred that it is not necessary to modify it. I will revert it back in the next commit.